examples/courses/2018-11-SenSys/sensys_udp_rx/main.c

```diff
@@ -84,16 +84,16 @@ int main(void) {
   ssize_t result = udp_recv(callback, packet_rx, MAX_RX_PACKET_LEN);
 
   switch (result) {
-    case TOCK_SUCCESS:
+    case RETURNCODE_SUCCESS:
       printf("Succesfully bound to socket, listening for UDP packets\n\n");
       break;
-    case TOCK_EINVAL:
+    case RETURNCODE_EINVAL:
       printf("The address requested is not a local interface\n");
       break;
-    case TOCK_EBUSY:
+    case RETURNCODE_EBUSY:
       printf("Another userland app has already bound to this addr/port\n");
       break;
-    case TOCK_ERESERVE:
+    case RETURNCODE_ERESERVE:
```

Comment:
For the courses, do we want to leave these as-they-were at the time of the course, or update them?

Reply:
I think we have to update them for build_all.sh to pass. Also, over time it seems like there is little value (other than for posterity, which git should already handle) to maintain guides for outdated versions of Tock. I'm all for moving to a model where all of our tutorials are always updated as tock is developed.